### ISSUE TITLE
IJulia installation Error on windows 10 using Julia 0.1.0 

### ISSUE THREAD

**Issue by Chetyh** [JuliaLang/IJulia.jl#741]
I have done these things before I added IJulia pakage: 
1. install Julia 1.0.0
2. install Avaconda with 3.6 python
3. open Julia repl and type
    ENV=["JUPYTER"]="C:\\Users\\sech1\\AppData\\Local\\Continuum\\Anaconda3\\Scripts\\jupyter.exe"
4. type using Pkg and Pkg.add("IJulia")， this is the second time I do so
   `julia> Pkg.add("IJulia")
  Updating registry at `C:\Users\Administrator\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
  Updating `C:\Users\Administrator\.julia\environments\v1.0\Project.toml`
 [no changes]
  Updating `C:\Users\Administrator\.julia\environments\v1.0\Manifest.toml`
 [no changes]`
5. type using IJulia, error occur
`[ Info: Precompiling IJulia [7073ff75-c697-5162-941a-fcdaad2a7d2a]
ERROR: LoadError: Blosc not installed properly, run Pkg.build("ZMQ"), restart Ju
lia and try again
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] top-level scope at C:\Users\Administrator\.julia\packages\ZMQ\ABGOx\src\ZMQ
.jl:14
 [3] include at .\boot.jl:317 [inlined]
 [4] include_relative(::Module, ::String) at .\loading.jl:1038
 [5] include(::Module, ::String) at .\sysimg.jl:29
 [6] top-level scope at none:2
 [7] eval at .\boot.jl:319 [inlined]
 [8] eval(::Expr) at .\client.jl:389
 [9] top-level scope at .\none:3
in expression starting at C:\Users\Administrator\.julia\packages\ZMQ\ABGOx\src\Z
MQ.jl:13
ERROR: LoadError: Failed to precompile ZMQ [c2297ded-f4af-51ae-bb23-16f91089e4e1
] to C:\Users\Administrator\.julia\compiled\v1.0\ZMQ\CaaO6.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] macro expansion at .\logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at .\loading.jl:1184
 [4] _require(::Base.PkgId) at .\logging.jl:311
 [5] require(::Base.PkgId) at .\loading.jl:852
 [6] macro expansion at .\logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at .\loading.jl:834
 [8] include at .\boot.jl:317 [inlined]
 [9] include_relative(::Module, ::String) at .\loading.jl:1038
 [10] include(::Module, ::String) at .\sysimg.jl:29
 [11] top-level scope at none:2
 [12] eval at .\boot.jl:319 [inlined]
 [13] eval(::Expr) at .\client.jl:389
 [14] top-level scope at .\none:3
in expression starting at C:\Users\Administrator\.julia\packages\IJulia\k5o7j\sr
c\IJulia.jl:36
ERROR: Failed to precompile IJulia [7073ff75-c697-5162-941a-fcdaad2a7d2a] to C:\
Users\Administrator\.julia\compiled\v1.0\IJulia\nfu7T.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] macro expansion at .\logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at .\loading.jl:1184
 [4] _require(::Base.PkgId) at .\logging.jl:311
 [5] require(::Base.PkgId) at .\loading.jl:852
 [6] macro expansion at .\logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at .\loading.jl:834`
I do not what error notice mean, it seems that I can not download something from the internet.

**Comment by Chetyh** [comment]
Correct, it has nothing to do with IJulia. I have fixed up this error.
Thanks.

Steven G. Johnson <notifications@github.com> 于 2018年9月8日 周六 21:19写道：

**Comment by Chetyh** [comment]
Yes, this is a download failure problem. I have already fixed up this
problem. Thanks a lot.

Tom Kwong <notifications@github.com> 于 2018年9月9日 周日 13:25写道：

**Comment by juanpassadore** [comment]
Hi. I am having a similar problem. https://www.dropbox.com/s/nq1l8rizeq2t3pi/Error_IJulia.png?dl=0. Working with Windows 7, and Julia 1.1.0. Thanks in advance.

**Comment by Chetyh** [comment]
> Hi. I am having a similar problem. https://www.dropbox.com/s/nq1l8rizeq2t3pi/Error_IJulia.png?dl=0. Working with Windows 7, and Julia 1.1.0. Thanks in advance.
Sorry, I do not remember how I solve it clearly. However, one thing I am sure of is that I update something of my windows 10 to fix it up.

**Comment by juanpassadore** [comment]
Ok, thanks, I will continue trying. Best, Juan


<http://economics.mit.edu/grad/juanpass>
